app/src/main/res/layout/fragment_contact_us.xml

```diff
@@ -0,0 +1,149 @@
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:id="@+id/constraintLayout_main"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:background="@color/n_60"
+    tools:context=".contactusform.ContactUsFragment">
+
+    <androidx.cardview.widget.CardView
+        android:id="@+id/card_view_layout"
+        android:layout_width="0dp"
+        style="@style/default_card_style"
+        app:layout_constraintBottom_toBottomOf="parent"
+        app:layout_constraintEnd_toEndOf="parent"
+        android:layout_height="wrap_content"
+        app:layout_constraintStart_toStartOf="parent">
+
+        <androidx.constraintlayout.widget.ConstraintLayout
+            android:id="@+id/constraintLayout_bottom"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
+            android:minHeight="500dp">
+
+            <ImageView
+                android:id="@+id/imageView_at_sign"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:layout_marginStart="@dimen/size_3x"
```

Comment:
Instead of adding for each View `android:layout_marginStart` you can just add padding/margin to parent View.

Reply:
When I use H2 class (following another comment) it has got its own padding and marginStart so I think I can't use global padding/margin this way. Am I right?

Reply:
@raaf1981 Correct, this class shouldn't have own padding. Let's leave at it is.